pkg/auth/github/config.go

```diff
@@ -11,9 +11,14 @@ type Config struct {
 	ClientSecret string
 	Organization string
 	Teams        string
+	Domain       string
 }
 
-func (c *Config) SetDefaults() {}
+func (c *Config) SetDefaults() {
+	if c.Domain == "" {
+		c.Domain = "github.com"
+	}
```

Comment:
As mentioned before, if the flag has a default value, this is no longer necessary.